static/app/views/projectInstall/issueAlertNotificationOptions.tsx

```diff
@@ -0,0 +1,150 @@
+import {useEffect, useMemo, useState} from 'react';
+import styled from '@emotion/styled';
+
+import MultipleCheckbox from 'sentry/components/forms/controls/multipleCheckbox';
+import {space} from 'sentry/styles/space';
+import {IssueAlertActionType} from 'sentry/types/alerts';
+import type {OrganizationIntegration} from 'sentry/types/integrations';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import SetupMessagingIntegrationButton, {
+  MessagingIntegrationAnalyticsView,
+} from 'sentry/views/alerts/rules/issue/setupMessagingIntegrationButton';
+import {useIssueAlertNotificationContext} from 'sentry/views/projectInstall/issueAlertNotificationContext';
+import MessagingIntegrationAlertRule from 'sentry/views/projectInstall/messagingIntegrationAlertRule';
+
+export const providerDetails = {
+  slack: {
+    name: 'Slack',
+    action: IssueAlertActionType.SLACK,
+    label: 'workspace to',
+    placeholder: 'channel, e.g. #critical',
+  },
+  discord: {
+    name: 'Discord',
+    action: IssueAlertActionType.DISCORD,
+    label: 'server in the channel',
+    placeholder: 'channel ID or URL',
+  },
+  msteams: {
+    name: 'MSTeams',
+    action: IssueAlertActionType.MS_TEAMS,
+    label: 'team to',
+    placeholder: 'channel ID',
+  },
+};
+
+enum MultipleCheckboxOptions {
```

Comment:
can we use a const enum here if we're not using the index
```suggestion
const enum MultipleCheckboxOptions {
```